source/Ship.cpp

```diff
@@ -3553,8 +3557,15 @@ int Ship::TakeDamage(vector<Visual> &visuals, const DamageDealt &damage, const G
 	// If this ship did not consider itself an enemy of the ship that hit it,
 	// it is now "provoked" against that government.
 	if(sourceGovernment && !sourceGovernment->IsEnemy(government)
-			&& (Shields() < .9 || Hull() < .9 || !personality.IsForbearing())
-			&& !personality.IsPacifist() && damage.GetWeapon().DoesDamage())
+			&& personality.IsForbearing() && !personality.IsPacifist()
+			&& (((damage.Shield() || damage.Discharge()) && Shields() < .9)
+			|| ((damage.Hull() || damage.Corrosion()) && Hull() < .9)
+			|| ((damage.Heat() || damage.Burn()) && isOverheated)
+			|| (damage.Ion() && CalculateJamChance(Energy(), ionization) > 0.1)
+			|| (damage.Slowing() && slowness > 10.)
+			|| ((damage.Energy() || damage.Ion()) && Energy() < 0.5)
+			|| ((damage.Fuel() || damage.Leak()) && fuel < JumpFuel() * 2.)
+			|| (damage.Disruption() && disruption > 100.)))
```

Comment:
`&& personality.IsForbearing()`? That doesn't look right.
```suggestion
			&& !personality.IsPacifist() && (!personality.IsForbearing()
				|| ((damage.Shield() || damage.Discharge()) && Shields() < .9)
				|| ((damage.Hull() || damage.Corrosion()) && Hull() < .9)
				|| ((damage.Heat() || damage.Burn()) && isOverheated)
				|| ((damage.Energy() || damage.Ion()) && Energy() < 0.5)
				|| ((damage.Fuel() || damage.Leak()) && fuel < JumpFuel() * 2.)
				|| (damage.Ion() && CalculateJamChance(Energy(), ionization) > 0.1)
				|| (damage.Slowing() && slowness > 10.)
				|| (damage.Disruption() && disruption > 100.)))
```

Reply:
~~Right I got my logic mixed up~~
I'm dyslexic so booleans are my worst enemies x)